### DISCUSSION TITLE
Nightscout/xDrip Google Domains (Other domains DDNS)

### DISCUSSION THREAD

**Discussion by bmpreston** [NightscoutFoundation/xDrip#2645] [Ideas]
Is it possible to allow other DDNS or specifically Google Domains to write to the file?  I'd like to SSL my Google Domains but the certbot is looking for the FreeDNS URL.  

Thanks!

**Comment by Navid200** [comment]
Our intent has been to create a free solution that can be used by everyone as easily as possible with minimum support requirement on our part.   That translates into choosing the best path and leaving alternative options alone.

There is no way we can include all options users may desire on the menus or create utilities for them or even test them.
If you like to use a private domain, you can.   But, you need to do it yourself.

So far, we have had a complaint with FreeDNS that some schools or work places have blocked access.  It is easy to address the problem by choosing a domain that is not very popular as explained here:  https://navid200.github.io/xDrip/docs/Nightscout/FD_Domains.html

Please let me know if you have tried FreeDNS and have experienced any problems.

**Reply by bmpreston** [reply]
I appreciate your quick and thorough response and understand the development dilemma..  Not what I was hoping for, understandable all the same.  Thanks again!  Great work and product!

**Reply by Navid200** [reply]
Do you mind sharing why FreeDNS does not work for you?

**Comment by bmpreston** [comment]
Of course.  It’s not that it doesn’t work for me, it’s that I prefer to
self host as much as possible, and therefore I’m trying to run on my own
domain and my own google domain.  I’m just not strong at all in nginx and
ubuntu//linux and can’t figure out how to letsencrypt my own domain on the
VM that is hosted.  When I run certbot, it wants to get a cert for the
freeddns (not my google domain) since I need to put my own domain in the
nginx conf file.

Thanks!
